app/layout/project/navigation/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+import { useRouter } from 'next/router';
+
+import { useProject } from 'hooks/projects';
+import { useScenario } from 'hooks/scenarios';
+
+import { cn } from 'utils/cn';
+
+export const Breadcrumbs = ({ className }: { className?: string }): JSX.Element => {
+  const { pathname, push, query } = useRouter();
+  const { pid, sid } = query as { pid: string; sid: string };
+  const projectQuery = useProject(pid);
+  const scenarioQuery = useScenario(sid);
+
+  return (
+    <nav className="flex font-sans text-xs text-white" aria-label="Breadcrumb">
```

Comment:
Why do you need a `font-sans` class at this point? You should have already inherited the font styles.